Llk/Sampler/Sampler.php

```diff
@@ -125,6 +131,35 @@ public function getCompiler()
     }
 
     /**
+     * Get a string representation of the current namespace skip token.
+     *
+     * @return  string
+     */
+    protected function getSkipToken()
```

Comment:
Either `getSkipToken` returns the AST of the skip token, or it returns a value for this token. Currently, it returns a value but it is not consistent with what the `generateToken` does.

So either:
* We change the name for `generateSkipToken`, or
* This methods an AST and we `visit` the AST in the `generateToken` methods.

I am ok with both. You have the choice if agree :smiley:!